package.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "fluidic-11ty",
-    "version": "0.1.0",
-    "description": "Fluid is an open, collaborative project to improve the user experience and inclusiveness of open source software.",
+    "name": "trivet",
+    "version": "0.1.0-dev.0",
```

Comment:
Any reason for the odd version number? Could this be 1.0.0 of trivet?

Reply:
@amb26 In my opinion it is still a prerelease — I would like to improve the included templates a bit before making it a 1.0.0. I tagged 0.1.0-0 so as to have something to publish to NPM (didn't want to lose trivet right after losing pekoe...).